src/tests/uat/nsx_remediation_uat.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020-2021. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""
+To execute, a profile must be provided using the standard CBC Credentials.
+
+The following API calls are tested in this script:
+
+Workload:
+* NSX Remediation
+  * https://developer.carbonblack.com/reference/carbon-black-cloud/workload-protection/latest/
+            appliance-service/#nsx-remediation
+"""
+
+import sys
+import random
+import time
+
+from cbc_sdk.helpers import build_cli_parser, get_cb_cloud_object
+from cbc_sdk.platform import Device
+from cbc_sdk.workload import NSXRemediationJob
+from cbc_sdk.errors import ApiError, ServerError, NSXJobError
+
+
+def get_devices(cb, ids):
+    """
+    Gets the devices that can be tested with NSX Remediation.
```

Comment:
Can you specify what indicates that a device can be tested?

Reply:
It's basically all devices for which `nsx_available` is `True`. That property being `True` means that the `deployment_type` is `WORKLOAD` and the `nsx_enabled` flag is set.  A comment has been added to that effect.